### DISCUSSION TITLE
Is it possible to use this with sonarr, radarr, etc.?

### DISCUSSION THREAD

**Discussion by srfsh** [porla/porla#325] [Q&A]
Hi,

I realized this project is very minimal and easier to extend, thus. I would like to use it with Sonarr, Radarr, etc. Is this possible? I've seen the pull request https://github.com/porla/porla/pull/271 regarding Sonarr, but I am not sure how to integrate. There's nothing documented regarding integration.

Thanks.

**Comment by vktr** [comment]
Hi!

It is possible right now with the [folder-monitor](https://github.com/porla-plugins/folder-monitor) plugin and the blackhole thing for Sonarr/Radarr.

I'm just starting to finish up the PR for Sonarr that adds Porla as an official supported client so until then the above your best bet!

**Reply by NLZ** [reply]
Hi vktr, any chance to resurrect this?
https://github.com/Sonarr/Sonarr/pull/6603
I think it would be a big step in enabling new users.
